Source/WebCore/rendering/RenderTextControl.cpp

```diff
@@ -96,6 +96,9 @@ RenderBox::LogicalExtentComputedValues RenderTextControl::computeLogicalHeight(L
     if (!innerText)
         return RenderBox::computeLogicalHeight(LayoutUnit(), LayoutUnit());
 
+    if (style().fieldSizing() == FieldSizing::Content)
+        return RenderBlockFlow::computeLogicalHeight(logicalHeight, logicalTop);
```

Comment:
`RenderBox::computeLogicalHeight` for consistency with the rest of the code?

Reply:
Does this work correctly if the textarea has a scrollbar?

Reply:
I used RenderBlockFlow because it's the parent to RenderTextControl but I doubt there's any difference in it. I'll double check it doesn't break anything and change that.

Also yes I believe it works fine with scrollbars will double check though.

Reply:
Changed to be RenderBox, I realised RenderBlockFlow and RenderBlock both don't override the function so its identical behaviour.

I've checked and the scrollbars work exactly as expected (and how CHromium behaves)